common/cmake/devices.cmake

```diff
@@ -12,7 +12,8 @@ function(DEFINE_ARCH)
   # ~~~
   # DEFINE_ARCH(
   #    ARCH <arch>
-  #    YOSYS_SCRIPT <yosys_script>
+  #    YOSYS_SYNTH_SCRIPT <yosys_script>
+  #    YOSYS_CONV_SCRIPT <yosys_script>
```

Comment:
Need to update documentation on these arguments

Reply:
Done.